gitlab/v4/objects/__init__.py

```diff
@@ -3601,7 +3601,7 @@ def save(self, **kwargs):
 
 
 class ProjectLabelManager(
-    ListMixin, CreateMixin, UpdateMixin, DeleteMixin, RESTManager
+    GetMixin, ListMixin, CreateMixin, UpdateMixin, DeleteMixin, RESTManager
```

Comment:
Thanks for this and nice catch! There's a convenience `RetrieveMixin` for get+list that we could also use here:

```suggestion
    RetrieveMixin, CreateMixin, UpdateMixin, DeleteMixin, RESTManager
```


Don't worry about the functional test failure, the environment needs a bit of cleanup. You could also add a `get()` test to the functional test suite after this section if you'd like:
https://github.com/python-gitlab/python-gitlab/blob/265dbbdd37af88395574564aeb3fd0350288a18c/tools/functional/api/test_projects.py#L141-L143

Reply:
Thanks! Changed.

Reply:
Thanks and sorry for the delay!